### PR TITLE
Addd Stych Scrypt plugin

### PR DESCRIPTION
**Pull request by rideam**

### CODE REVIEW

File: Password Hashing Plugins/src/main/java/com/mycompany/fusionauth/plugins/guice/MyExampleFusionAuthPluginModule.java [hunk 1]
```diff
@@ -71,16 +72,10 @@ protected void configure() {
...
     // passwordEncryptorMapBinder.addBinding("example-argon2id").to(ExampleArgon2idPasswordEncryptor.class);
 
-    // PBDKF2HMACSHA256 with Keylength 512, useful for keycloak
     passwordEncryptorMapBinder.addBinding("example-keycloak").to(ExamplePBDKF2HMACSHA256KeyLength512PasswordHasher.class);
-
-    // Example PBKDF2 with a SHA-1
     passwordEncryptorMapBinder.addBinding("example-salted-pbkdf2-hmac-sha1-10000").to(ExamplePBDKF2HMACSHA1PasswordEncryptor.class);
-
-    // Example Firebase Scrypt 
-    passwordEncryptorMapBinder.addBinding("example-salted-firebase-scrypt").to(ExampleFirebaseScryptPasswordEncryptor.class);
```
Comment (mooreds) [inline Password Hashing Plugins/src/main/java/com/mycompany/fusionauth/plugins/guice/MyExampleFusionAuthPluginModule.java:81]: Why did you remove this?

Comment (rideam) [reply Password Hashing Plugins/src/main/java/com/mycompany/fusionauth/plugins/guice/MyExampleFusionAuthPluginModule.java:81]: Sorry during the modifications some of the comments were removed which moved the code lines to different line numbers but otherwise the actual code line was not removed. I have fixed this and put back the comments.

**Comment by mooreds** [comment]
Is this ready for review?

**Review by mark-robustelli — APPROVED** [review]
